oleesir/store-manager
# ft(sales-record):implement ft-sale-records-template

#### What does this PR do?
Implement sale-records template
#### Description of Task to be completed?
Implement sale-records template
#### How should this be manually tested?
* Navigate to the UI folder and click on sale-records.html in your browser
* You will see sale-records page in template
#### What are the relevant pivotal tracker stories?
[#161193052](https://www.pivotaltracker.com/story/show/161193052)
#### Any background context you want to add?
N/A

#### Screenshots
<img width="1280" alt="screen shot 2018-10-12 at 9 58 15 pm" src="https://user-images.githubusercontent.com/17679511/46893754-f4474700-ce69-11e8-99ab-212cb4366399.png">
